lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -78,6 +78,10 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
       canvas.saveLayer(Rect.fromLTWH(0, -40, size.width + 40, size.height + 40), Paint());
     }
 
+    for(BetweenBarsData betweenBarsData in data.betweenBarsData) {
+      drawBetweenBarsArea(canvas, size, data.lineBarsData[betweenBarsData.fromIndex], data.lineBarsData[betweenBarsData.toIndex], betweenBarsData);
```

Comment:
We can have this code `data.lineBarsData[betweenBarsData.fromIndex]` inside the function, instead of passing it to function, because you are passing `betweenBarsData`, then we can retrieve these two arguments from it, as you know we should not pass lots of arguments to a function, it makes complication.